pages/dev-snippets/categories/index.md

```diff
@@ -110,7 +110,8 @@ Alternative
 	// Check if the user is browsing a category
 	if ($WHERE_AM_I=='category') {
 		// Get the category key from the URL
-		$categoryKey = $Url->slug();
+		global $url;
```

Comment:
Hi, this is not correct for the documentation because in you example you are calling `$url` from inside some function so you need to do this global. Maybe we can include a note instead of modified the code.
What is wrong is the uppercase in `$Url`, need to be `$url`.

Reply:
oh sorry, I just used the snippet and it only worked with calling it and thought maybe this was missed here as well. The uppercase is fine I guess, I just used a theme template and lowercase was used. Sorry to bother, you can close it